services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/impl/DpmServiceImpl.java

```diff
@@ -408,6 +491,10 @@ private void patchGoalstateForNeighbor(NetworkConfiguration networkConfig, Unica
             }
         }
 
+        if (patchRouterSubnetStates) {
+            patchGoalstateForRouterSubnet(networkConfig, hostsSubnets, multicastGoalState);
```

Comment:
Pls make a comment pointing to the Issue page for tracking purpose.

Reply:
comment added